src/test/codegen/cf-protection.rs

```diff
@@ -0,0 +1,33 @@
+// Test that the correct module flags are emitted with different control-flow protection flags.
+
+// revisions: undefined none branch return full
+// needs-llvm-components: x86
+// [undefined] compile-flags:
+// [none] compile-flags: -Z cf-protection=none
+// [branch] compile-flags: -Z cf-protection=branch
+// [return] compile-flags: -Z cf-protection=return
+// [full] compile-flags: -Z cf-protection=full
+// compile-flags: --target x86_64-unknown-linux-gnu
+
+#![crate_type = "lib"]
```

Comment:
This needs to be a `#[no_core]` test. There are a number of others that use `#[no_core]` that you can use as an example.

Reply:
Also, what is the correct state to put this PR in? Should I give another rustbot command to change the label? Or do you?